cfme/tests/ansible/test_embedded_ansible_tags.py

```diff
@@ -60,88 +62,33 @@ def _check_tag_place(item, tag_place):
     return _check_tag_place
 
 
-@pytest.mark.parametrize('tag_place', [True, False], ids=['details', 'list'])
-def test_tag_ansible_repository(ansible_repository, tag_place, check_tag_place):
-    """ Test for cloud items tagging action from list and details pages
-
-    Polarion:
-        assignee: anikifor
-        initialEstimate: 1/4h
-        casecomponent: Tagging
-    """
-    check_tag_place(ansible_repository, tag_place)
-
-
-@pytest.mark.parametrize('tag_place', [True, False], ids=['details', 'list'])
-def test_tag_ansible_credential(credential, tag_place, check_tag_place):
-    """ Test for cloud items tagging action from list and details pages
-
-    Polarion:
-        assignee: anikifor
-        initialEstimate: 1/4h
-        casecomponent: Tagging
-    """
-    check_tag_place(credential, tag_place)
-
-
-@pytest.mark.parametrize('tag_place', [True, False], ids=['details', 'list'])
-def test_tag_ansible_playbook(playbook, tag_place, check_tag_place):
-    """ Test for cloud items tagging action from list and details pages
-
-    Polarion:
-        assignee: anikifor
-        initialEstimate: 1/4h
-        casecomponent: Tagging
-    """
-    check_tag_place(playbook, tag_place)
-
-
-@pytest.mark.meta(automates=[1526217])
-@pytest.mark.parametrize('visibility', [True, False], ids=['visible', 'notVisible'])
-def test_tagvis_ansible_repository(ansible_repository, check_item_visibility, visibility):
-    """ Test for cloud items tagging action from list and details pages
+@pytest.fixture(params=['credential', 'ansible_repository', 'playbook'])
+def obj(request):
+    return request.getfixturevalue(request.param)
 
-    Bugzilla:
-        1526217
 
-    Polarion:
-        assignee: anikifor
-        initialEstimate: 1/4h
-        casecomponent: Tagging
-    """
-    check_item_visibility(ansible_repository, visibility)
+@pytest.fixture(params=['check_tag_place', 'check_item_visibility'], ids=['tag_details', 'tagvis'])
+def ansible_tag_test_func(request, obj):
+    test_func = request.getfixturevalue(request.param)
+    return partial(test_func, obj)
 
 
-@pytest.mark.meta(automates=[1526219])
+@pytest.mark.meta(automates=[1526219, 1526217, 1526218])
 @pytest.mark.parametrize('visibility', [True, False], ids=['visible', 'notVisible'])
-def test_tagvis_ansible_credential(credential, check_item_visibility, visibility):
+def test_ansible_(ansible_tag_test_func, visibility):
```

Comment:
Though its an 'abstract' test function, I think it should have a complete name. `test_ansible_tagging`